README.md

```diff
@@ -1,55 +1,149 @@
-# COVID-19 CLI
+# CoVID-19 CLI
 
 <img src="https://badges.frapsoft.com/os/v1/open-source.png?v=103"> <a href="LICENSE"><img alt="GitHub" src="https://img.shields.io/github/license/dmahajan980/covid19"></a>
 
 A command-line interface to check the current statistics of COVID-19 of the world or a particular country.
 
-## TABLE OF CONTENTS
-1. [Getting Started](#getting-started)
-2. [Usage](#usage)
-3. [Contributing](#contributing)
-5. [License](#license)
+## Table of Contents
 
-## GETTING STARTED
+1.  [Getting Started](#getting-started)
+2.  [Usage](#usage)
+3.  [Contributing](#contributing")
+4.  [License](#license)
 
-1. Clone or download the repository.
+## Getting Started
 
-2. Install all the dependencies using the following command
+To get started with the CLI is very simple
```

Comment:
I think you should change this to something like, "Getting started with the CoVID-19 CLI is very simple. Kindly follow the steps given below"